tests/unit/test-util/test.desktop-notifier.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+import {it, describe} from 'mocha';
+import {assert} from 'chai';
+import sinon from 'sinon';
+
+import {desktopNotifications} from '../../../src/util/desktop-notifier';
+import {createLogger} from '../../../src/util/logger';
+
+describe('util/desktop-notifier', () => {
+  describe('desktopNotifications()', () => {
+    const expectedNotification = {
+      title: 'web-ext run: title',
+      message: 'message',
+    };
+
+    it('is called and creates a message with correct parameters', () => {
+      const fakeNotifier = {
+        notify: sinon.spy(() => Promise.resolve()),
```

Comment:
The signature that you need to mock looks more like this (according to their docs):
````javascript
const fakeNotifier = {
  notify: sinon.spy((options, callback) => callback()),
};
````